Getting-Started/git.md

```diff
@@ -1,28 +1,145 @@
-# git
+# PGG's Git Crash Course: How to 'git' good
 
-## What is it
-Git is software for tracking changes in any set of files, usually used for coordinating work among programmers collaboratively developing source code. (Wikipedia)
+~Christian Peters
 
-[GitHub git cheat sheet](https://training.github.com/downloads/github-git-cheat-sheet/)
+## What is git?
 
-## Example workflows
+Something that is all too common with us nerds who work with computers is that we make mistakes. _A LOT_. In fact, it wouldn't be too far off to define working with software "a constant evolving problem that never goes away". However, when you solve that problem, it's all the more sweet.
 
-* Clone a repo: `git clone [url]`
-* Make some changes (after you have edited files or folders):
-  * Stage every altered file for commit: `git add [path to changed/new files and folders]`
-  * Create a commit: `git commit`
-  * Push all local commits to the remote repository: `git push`
-* Branching
-  * Create a branch: `?`
-  * Checkout/switch to a branch: `?`
-  
+But when you're working with code that is also constantly evolving and changing, it's important to keep track of your work. Everyone has been working on a coding project, added a function and then had everything break. I know I've been there. When that happened, I would have killed to have something that would have kept track of my routine "backups" of my code. Enter git.
 
-## Branching strategies
-Really you just need to understand [A successful Git brnaching model](https://nvie.com/posts/a-successful-git-branching-model/).
+As we have referenced a lot in this guide, let's see what Dr. Wikipedia has to say:
 
-## [Issues](https://guides.github.com/features/issues/)
-Use issues to track ideas, enhancements, tasks, or bugs for work on GitHub.
+> Git is software for tracking changes in any set of files, usually used for coordinating work among programmers collaboratively developing source code. ([Wikipedia](https://en.wikipedia.org/wiki/Git))
 
-## Project Boards
-Project boards on GitHub help you organize and prioritize your work. You can create project boards for specific feature work, comprehensive roadmaps, or even release checklists. With project boards, you have the flexibility to create customized workflows that suit your needs.
+Basically, to boil down the boiled down version of git, it is a complex remote integrated system restore. If you've ever used Windows, you know that making a system restore point enables your computer to completely revert back to that certain point when something bad happened. You downloaded a virus from a Minecraft mod website, but never fear, Windows made a restore point to save your computer and act like that malicious software never existed. This is the same concept that is applied to git, but with code.
```

Comment:
do not like windows system restore analogy

Reply:
Changed to: "Basically, to boil down the boiled down version of git, it is a version-control system that keeps track of changes to files. You then can use this to coordinate and collaborate between multiple people."